libbeat/processors/add_kubernetes_metadata/kubernetes.go

```diff
@@ -26,17 +26,20 @@ import (
 	"github.com/elastic/beats/libbeat/common/kubernetes"
 	"github.com/elastic/beats/libbeat/logp"
 	"github.com/elastic/beats/libbeat/processors"
+
+	k8s "k8s.io/client-go/kubernetes"
```

Comment:
This import should be placed before beats imports.
```
	"fmt"
	"time"

	k8s "k8s.io/client-go/kubernetes"

	"github.com/elastic/beats/libbeat/beat"
	"github.com/elastic/beats/libbeat/common"
	"github.com/elastic/beats/libbeat/common/kubernetes"
	"github.com/elastic/beats/libbeat/logp"
	"github.com/elastic/beats/libbeat/processors"
```

And I wonder if we should use another alias, `k8s` is basically the same as `kubernetes`, maybe `k8sclient`... Not sure.